### PR TITLE
GRID-303: Show drill-down controls in its own column

### PR DESCRIPTION
**Pull request by joneit**

### CODE REVIEW

**Comment by joneit** [comment]
@Dwaynekj This is mostly sample code for WIlliam, with a couple of hypergrid tweaks to make it feasible.

File: src/features/ColumnSorting.js [hunk 1]
```diff
@@ -32,8 +37,12 @@ var ColumnSorting = Feature.extend('ColumnSorting', {
      * @param {Object} event - the event details
      */
     handleMouseMove: function(grid, event) {
-        var y = event.gridCell.y;
-        if (this.isFixedRow(grid, event) && y < 1) {
+        var gridCell = event.gridCell;
+        if (
```
Comment (Dwaynekj) [inline src/features/ColumnSorting.js]: What the purpose for this?

Comment (joneit) [inline src/features/ColumnSorting.js]: This code originally changed the mouse pointer to a "pointer" cursor (index finger) when hovering over a header cell (header row is currently always row 0, aka < 1). As I showed you, I added a line to this code to also check that the column's `unsortable` (new) property is falsy.

**Comment by Dwaynekj** [comment]
👍

**Pull request merged**